### PR TITLE
feat: CHK-2059 configuration for PROD env

### PR DESCRIPTION
**Pull request by Fernando-Granato**
#### List of Changes
Added stage into deploy pipeline for PROD env.
Added helm value file for PROD env.

#### Motivation and Context
The change was made to provide for release in PROD of ACA.

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as
  expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: .devops/deploy-pipelines.yml [hunk 1]
```diff
@@ -265,13 +258,13 @@ stages:
                   semver: '${{ parameters.RELEASE_SEMVER }}'
           - template: azure-templates/chart-current-version.yml
 
-  - stage: "Release_tag_docker"
-    displayName: 'Docker tag release'
+  - stage: "tag_docker_release"
+    displayName: 'Tag Docker image to be release'
```
Comment (giovanniberti) [inline .devops/deploy-pipelines.yml:262]: grammar
```suggestion
    displayName: 'Tag Docker image to be released'
```

File: .devops/deploy-pipelines.yml [hunk 2]
```diff
@@ -295,10 +288,17 @@ stages:
...
-    ${{ if eq(parameters['SKIP_RELEASE'], True) }}:
-      dependsOn: Release
-    ${{ if eq(parameters['SKIP_RELEASE'], False) }}:
-      dependsOn: [ Release_tag_docker,Release ]
+    dependsOn: [ tag_docker_release,Release ]
+    condition: |
+      and(
+        eq(${{parameters.UAT_PROD_DEPLOY}}, true),
+        in(dependencies.tag_docker_release.result, 'Succeeded', 'Skipped'),
+        in(dependencies.Release.result, 'Succeeded', 'Skipped'),
```
Comment (giovanniberti) [inline .devops/deploy-pipelines.yml:296]: I'd replace these with `ne(..., 'Failed')`

File: .devops/deploy-pipelines.yml [hunk 3]
```diff
@@ -336,6 +336,7 @@ stages:
   - stage: "Green_API_UAT_test"
     displayName: 'Running API test on green app'
     dependsOn: Deploy_UAT_Green
+    condition: eq(dependencies.Deploy_UAT_Green.result, 'Succeeded')
```
Comment (giovanniberti) [inline .devops/deploy-pipelines.yml:339]: Isn't this redundant with the previous line?

**Review by giovanniberti — APPROVED** [review]
Left some minor comments, LGTM

File: .devops/deploy-pipelines.yml [hunk 4]
```diff
@@ -344,4 +345,266 @@ stages:
...
+
+  # --- START Deploy PROD --- #
+  - stage: "Prod_WaitForApproval"
+    displayName: 'PROD approval deployment'
+    dependsOn: [ Deploy_UAT_Green,Release ]
+    condition: |
+      and(
+        eq(${{parameters.UAT_PROD_DEPLOY}}, true),
+        eq(dependencies.Deploy_UAT_Green.result, 'Succeeded'),
+        in(dependencies.Release.result, 'Succeeded', 'Skipped')
```
[the remark this answers is not in the document]
Comment (Fernando-Granato) [reply .devops/deploy-pipelines.yml]: Fixed with b379d7460a95d10f31e811f8276a6bf4c7e2a3ab

File: .devops/deploy-pipelines.yml (hunk 3 above)
Comment (Fernando-Granato) [reply .devops/deploy-pipelines.yml]: Fixed with b379d7460a95d10f31e811f8276a6bf4c7e2a3ab

File: .devops/deploy-pipelines.yml (hunk 2 above)
Comment (Fernando-Granato) [reply .devops/deploy-pipelines.yml]: Fixed with b379d7460a95d10f31e811f8276a6bf4c7e2a3ab

**Review by pietro-tota — APPROVED** [review]

**Pull request merged**